packages/@react-aria/tag/src/useTag.ts

```diff
@@ -69,7 +68,7 @@ export function useTag(props: TagProps<any>, state: GridState<any, any>): TagAri
   let domProps = filterDOMProps(props);
   return {
     clearButtonProps: mergeProps(pressProps, {
-      'aria-label': removeString,
+      'aria-label': formatMessage('remove', {label: item.textValue || children}),
```

Comment:
Why is this necessary? Aren't we already using `aria-labelledby` to get the label text included here?

Reply:
This was an effort to improve TalkBack's announcement of the clear button where it would just say "[TAG NAME] remove" making it not very clear that remove wasn't part of the tag's name. I'm fine punting this for later, see https://github.com/adobe/react-spectrum/pull/2815#discussion_r801141239 for the messaging I think would make more sense

Reply:
Oh. Would reordering the ids in the aria-labelledby help so it says "Remove [TAG NAME]" instead?